### DISCUSSION TITLE
프리텐다드와 무슨차이가 있는거죠..?

### DISCUSSION THREAD

**Discussion by hi-one-gg** [wanteddev/wanted-sans#18] [Q&A] [answered]
프리텐다드에 꽂혀서 무조건 다 프리텐다드로 쓰는 사람인데 올만에 들어왔떠니 원티드 폰트가 나왔네요? 기쁜맘으로 다운받아서 비교해보는데 프리텐다드랑 100%일치하는데 어떤 차이가 있나요?

**Comment by orioncactus** [comment, marked as answer]
안녕하세요! 매우 늦게 답장드려 죄송합니다 😅 이제야 확인해 답장드립니다!

원티드 산스는 프리텐다드와 같은 폰트 높이를 가지고 있고, 한글 모양이 프리텐다드와 (사실상) 같기 때문에 한글만 쓰면 큰 차이가 없으나, 라틴이나 숫자에서 그 차이를 도드라지게 보실 수 있는데요, 스타일적으로 보면 네오그로테스크에 있는 프리텐다드와 다르게 원티드 산스는 한글을 제외한 부분이 지오메트릭과 휴머니스트, 네오그로테스크 중간 즈음에 있는 것이 차이점입니다. 쉽게 이야기하면 원티드 산스는 강직하면서도 유연한 글꼴로 보여지도록 했습니다.

자세한 글꼴에 대한 설명은 [/documentation/concept](https://github.com/wanteddev/wanted-sans/blob/HEAD/documentation/concept/README.md)에, 그리고 기능에 대한 설명은 [/documentation/feature](https://github.com/wanteddev/wanted-sans/tree/main/documentation/features)에 적어두었으니, 시간 되실 때 한번 둘러보시고, 궁금하신 게 있다면 언제든지 알려주세요! ☺️ 이어서 이 이슈는 Discussions에 가까워서, Discussions 글로 바꿔두겠습니다.
